docs/contributors.md

```diff
@@ -1,40 +1,113 @@
+
+- Maintainers will review your pull request
+- If a maintainer requests changes, first of all, try to think about this request critically and only after that implement and request another review
+- If your PR gets accepted, it will soon be merged into the main branch. But your contribution will take effect only after the release of a new version of the action
+> Sometimes maintainers reject pull requests and that's ok! Usually, along with rejection, we supply the reason for it. Nonetheless, we still really appreciate you taking the time to do it, and we don't take that lightly :heart:
+
+## Contribute code
+
+We like code commits a lot! They're super handy, and they keep the project going and doing the work it needs to do to be useful to others.
+
+Code contributions of just about any size are acceptable!
+
+The main difference between code contributions and documentation contributions is that contributing code requires the inclusion of relevant tests for the code being added or changed. Contributions without accompanying tests will be held off until a test is added unless the maintainers consider the specific tests to be either impossible or way too much of a burden for such a contribution.
+
+**How to contribute:**
+
+Pull requests are the easiest way to contribute changes to git repos at GitHub. They are the preferred contribution method, as they offer a nice way of commenting and amending the proposed changes.
+
+- Please check that no one else has already created a pull request with these changes
+- Use a "feature branch" for your changes. That separates the changes in the pull request from your other changes and makes it easy to edit/amend commits in the pull request
+-  Format, build and test changes
+- Make sure your changes are well formatted and that all tests are passing
+- If your pull request is connected to an open issue, please, leave a link to this issue in the `Related issue:` section
+- If you later need to add new commits to the pull request, you can simply commit the changes to the local branch and then push them. The pull request gets automatically updated
+
+**Learn more about how to work with the repository:**
+
+- To implement new features or fix bugs, you need to make changes to the `.ts` files, which are located in the `src` folder
+- To comply with the code style, **you need to run the `format` script**
+- To transpile source code to `javascript` we use [NCC](https://github.com/vercel/ncc). **It is very important to run the `build` script after making changes**, otherwise your changes will not get into the final `javascript` build
+
+**Learn more about how to implement tests:**
+
+Adding or changing tests is an integral part of making a change to the code. 
+Unit tests are in the `__tests__` folder, and end-to-end tests are in the `workflows` folder (in particular, in the file [test-dotnet.yml](https://github.com/actions/setup-dotnet/blob/main/.github/workflows/test-dotnet.yml).
```

Comment:
`in particular, in the file` to `in the particular file`

Reply:
I would go with "to be exact, in the file ..."